src/components/Grid/Grid.story.jsx

```diff
@@ -29,45 +29,47 @@ stories.add(
             ))}
           </GridContainer>
         </section>
-        <section className="show-in-tests" style={{ width: 1280 }}>
-          <GridContainer className="border bg-teal mb-20">
-            {range(1, 13).map(() => (
-              <div className="cell-1 row-1 h-12 text-black text-center border bg-white">
-                <p className="mt-0 pt-12 text-xs">60px</p>
-              </div>
-            ))}
-            {range(1, 13).map(() => (
-              <div className="cell-1 row-2 h-12 text-black text-center border bg-white">
-                <p className="mt-0 pt-12 text-xs">60px</p>
-              </div>
-            ))}
-          </GridContainer>
-          <GridContainer className="border bg-teal mb-20" fullWidth>
-            {range(1, 13).map(() => (
-              <div className="cell-1 row-1 h-12 text-black text-center border bg-white">
-                <p className="mt-0 pt-12 text-xs">60px</p>
-              </div>
-            ))}
-            {range(1, 13).map(() => (
-              <div className="cell-1 row-2 h-12 text-black text-center border bg-white">
-                <p className="mt-0 pt-12 text-xs">60px</p>
-              </div>
-            ))}
-          </GridContainer>
+        {process.env.STORYBOOK_PERCY_ENV ? (
+          <section className="show-in-tests" style={{ width: 1280 }}>
```

Comment:
Do we still need the `show-in-tests` if we are using the conditional on the env?

Reply:
not necessarily, but can't hurt as a fail safe